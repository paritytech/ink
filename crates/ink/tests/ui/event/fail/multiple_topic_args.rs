#[ink::event]
pub struct Event {
    #[ink(topic)]
    #[ink(topic)]
    pub topic: [u8; 32],
}

fn main() {}