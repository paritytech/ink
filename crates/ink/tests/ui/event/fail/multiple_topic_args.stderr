error: Only a single `#[ink(topic)]` attribute allowed.
 --> tests/ui/event/fail/multiple_topic_args.rs:4:5
  |
4 |     #[ink(topic)]
  |     ^
